features/posting_a_task.feature

```diff
@@ -119,6 +119,18 @@ Feature: Posting a job
     And I have generated a job in the Job Store
     When I call POST /jobs/{id}/tasks using the generated id
     """
-    { "task_name": "florence", "number_of_documents": 29
+    { "task_name": "dataset-api", "number_of_documents": 29
     """
     Then the HTTP status code should be "401"
+
+  Scenario: Invalid task name returns bad request error
+
+    Given I use a service auth token "validServiceAuthToken"
+    And zebedee recognises the service auth token as valid
+    And I have generated a job in the Job Store
+    And I call POST /jobs/{id}/tasks using the generated id
+    """
+    { "task_name": "florence", "number_of_documents": 29 }
+    """
+    Then the HTTP status code should be "400"
+    
```

Comment:
Looks like something strange at the end of the file (stray space?)

Reply:
I normally put a new line at the end of each file (or sometimes the formatter does it for me). Should I not do that here?

Reply:
The newline is fine but it looked like there was a space on the newline or something?